AirportTicketBookingTest/Flights Tests/FilterFlightsTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AirportTicketBooking;
+using AirportTicketBooking.Enum;
+using AirportTicketBooking.Repositories;
+using Moq;
+
+namespace AirportTicketBookingTest.Flights_Tests
+{
+    public class FilterFlightsTests
+    {
+        [Theory]
+        [InlineData("Flightnum", "FL456")]
+        [InlineData("NumberOfSeats", 200)]
+        [InlineData("DepartureAirport", "LHR")]
+        public void Test_FilterBookings_MultipleFilters(string filterPropertyName, object filterValue)
```

Comment:
Let's have a name like
SERVICENAME_WhatYouExpectItToDo_When?
cause I need to know we test what? when and why? so if any test become failed I'll know which one and which scenario exactly
like DivisionService_ShouldThrowDivideByZeroException_WhenDivideByZero
and let's apply it for all tests